docs/account/emails.md

```diff
@@ -2,12 +2,41 @@
 
 To manage the email addresses associated with your account and your email notifications, click on your avatar on the top right-hand corner and open **Your account**, page **Emails**.
 
-## Updating your email addresses
+![Email settings](images/emails-notifications.png)
+
+## Updating your email addresses {: id="updating"}
+
+Codacy pulls email addresses from the Git provider associated with your current session. On the **Emails** page, you can verify which email addresses are associated with your Codacy account.
+
+To update the email addresses associated with your Codacy account, do the following:
 
-Codacy pulls email addresses from the Git provider associated with your current session. To update the email addresses displayed on Codacy, update them on your Git provider, then log out and back in to Codacy.
+1.  Update your email addresses on your Git provider ([GitHub](https://docs.github.com/en/enterprise-cloud@latest/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/adding-an-email-address-to-your-github-account), [Bitbucket](https://support.atlassian.com/bitbucket-cloud/docs/set-email-aliases/), or [GitLab](https://docs.gitlab.com/ee/user/profile/#add-emails-to-your-user-profile)).
+
+1.  Log out and log in to Codacy again.
```

Comment:
I suggest a tweak to align the wording with the sentence after this one on [L18](https://github.com/codacy/docs/pull/1806/files#diff-56b412633cbfb63c07e4248554092c225fd2b007ce19fff5fc401d592c6bd4e4R18):

```suggestion
1.  Log out and log back in to Codacy.
```